cli/src/pcluster/config/update_policy.py

```diff
@@ -282,6 +282,25 @@ def condition_checker_queue_update_strategy(change, patch):
     return result
 
 
+def condition_checker_queue_update_strategy_on_remove(change, patch):
+    result = not patch.cluster.has_running_capacity()
+    # Update of list element value is possible if one of the following is verified:
+    # - fleet is stopped
+    # - queue update strategy is set (different from default)
+    # - new list element is added
+    if is_slurm_queues_change(change):
+        result = (
+            result
+            or patch.target_config.get("Scheduling")
+            .get("SlurmSettings", {})
+            .get("QueueUpdateStrategy", QueueUpdateStrategy.COMPUTE_FLEET_STOP.value)
+            != QueueUpdateStrategy.COMPUTE_FLEET_STOP.value
+            or all(value in change.new_value for value in change.old_value)
```

Comment:
We should also check if `change.is_list` is `True`

Reply:
`change.is_list` is `False` in this case. I think because only nested section could be considered as list, and not actual values of type list. This is an implementation detail of the Change object

Reply:
added a check on the type of `change.old_value`